routed-js/Format.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
```

Comment:
Note fore reviewer: all `routed-js/*.js` files are generated by TypeScript compiler, so there are no a lot of sense to review it.